### PR TITLE
Dandelion 'es' Translation

### PR DESCRIPTION
**Pull request by lh1008**
Hey everyone,

Adding 'Dandelion++' (https://www.monerooutreach.org/stories/dandelion.html) Spanish version.

Pinging @BlackLotus64 @BTCLovera for review

Thank you :).

### CODE REVIEW

**Review by erciccione — COMMENTED** [review]
The English version has been edited. Is that on purpose?

File: monero-outreach-docs/translations/es/archive/dandelion_es.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+[4] Chainalysis, [chainalysis.com](https://www.chainalysis.com/).
+[5] Adding Dandelion++ support to public networks: #6314, [github.com/monero-project/monero/pull/6314](https://github.com/monero-project/monero/pull/6314).  
+
+### Resumen
+
+Un poderoso método para prevenir la desanonimización en Monero está por venir, Dandelion++. Este método desarrollado originalmente para Bitcoin y aplicado a Monero por el desarrollador Lee Clagett, previene que adversarios conecten las direcciones IP a las transacciones de Monero, incluso en ataques sofisticados a gran escala. Se enfoca particularmente en la resistencia a botnets. Dandelion++ usa las fases de tallo y cipsela haciendo que la información tome forma de diente de león o dandelion en inglés, y por eso su nombre. Esta nueva característica mejorará las ya existentes características de privacidad de Monero y llevar a Moneroen un camino de progreso continuo.
```
Comment (BTCLovera) [inline monero-outreach-docs/translations/es/archive/dandelion_es.md:64]: y llevar a Moneroen un camino de progreso continuo..... "llevar a Monero en"

**Review by BTCLovera — COMMENTED** [review]
Good translation, I did not find errors

File: monero-outreach-docs/translations/es/archive/dandelion_es.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/dandelion_es.md]: Fixed, thank you :)

**Comment by lh1008** [comment]
Hey @BTCLovera I did the suggested fix. Could you please approve the Pull Request?

**Review by erciccione — COMMENTED** [review]
@lh1008 this PR will need to have only two commits. One related to the changes made to the original file and one for the spanish translation.

**Review by BTCLovera — APPROVED** [review]

**Comment by lh1008** [comment]
@erciccione done (2 commits). 

I thought I had answered your comment before but I didn't:

> The English version has been edited. Is that on purpose?

Yes, it's un purpose, I wanted to take advantage I was making this translation to update the dandelion_en file and not open another Pull Request for it.

**Review by erciccione — CHANGES_REQUESTED** [review]
Please make one commit for the changes to the Spanish file and one for the English file, like i asked in my previous comment. There is no point in having one commit for the spanish file and one for the review of spanish and the edits to the file in english.

**Comment by lh1008** [comment]
@erciccione done

**Comment by erciccione** [comment]
Nope. You still have changes to both files in one of the commits. Follow these steps (from the root folder):

1. `git reset HEAD~2`
2. `git commit en/archive/dandelion_en.md -m "Updated dandelion_en file"`
3. `git commit translations/es/archive/dandelion_es.md -m "Added 'es' dandelion file"`
4. `git push -f origin dandelion_es`

**Comment by lh1008** [comment]
@erciccione 

image

**Comment by lh1008** [comment]
What is root folder? master?

**Comment by lh1008** [comment]
@erciccione I think I did it this time. Please let me know :)

**Review by erciccione — APPROVED** [review]

**Pull request merged**